xAPI.md

```diff
@@ -507,6 +507,45 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of a series of systems.  The cloud in the picture 
+represents a Learning Management System, or LMS.  An LMS, if present, would likely encapsulate everything within 
+the cloud.  In either case, a user will be dependent on a relationship between content and one or more Learning 
+Record Stores.  It is also possible a learner is dependent on an Activity Provider to deliver and/or authorize use 
+of content to them.  Once this triangle of authorization and access is complete, a registration exists for that 
+user in that content for that LRS.  LRSs, at request of the user, can also share data to other LRSs that may or 
+may not be a part of the LMS framework.  Web Services, again as a part of or independent of the LMS, can pull 
+data from one or more LRSs, send them through other Web Services or LRSs.
+
+##### Activity Data and Metadata
+
+![xAPI Activity Data and Metadata](/xAPIMetadata.jpg)
```

Comment:
I'm doing better with this one, but there's also a lot I don't get. For example, an LRS usually *won't* contain anything you'd call a "canonical statement", since a canonical statement takes the actual statement sent and combines it with the current 'canonical' versions of the activities and agents and verbs in it, meaning it would be a pretty bad idea for the LRS to store that form directly. Also, the "Metadata consumer" isn't consuming metadata, but actual data, if they're reading from the statements endpoint. And, as basically no one is using Activity IDs that resolve to actual activity metadata, by far the most common way for a 'canonical activity' to be populated is based on other statements, which I'm not even sure the diagram includes as an option.